Xamarin.Forms.Core/Label.cs

```diff
@@ -215,13 +222,31 @@ public Thickness Padding
 			get { return (Thickness)GetValue(PaddingProperty); }
 			set { SetValue(PaddingProperty, value); }
 		}
-		
+
 		public TextType TextType
 		{
 			get => (TextType)GetValue(TextTypeProperty);
 			set => SetValue(TextTypeProperty, value);
 		}
 
+		public AutoFitTextMode AutoFitText
+		{
+			get => (AutoFitTextMode)GetValue(AutoFitTextProperty);
+			set => SetValue(AutoFitTextProperty, value);
+		}
+
+		public int MinFontSize
```

Comment:
The only thing I'm wondering is if we should name this to be more cohesive to `AutoFitText`. Right now it might not be apparent that these work together.

Reply:
yeah...
I was thinking about this, and does not have any other feature that can use Max/MinFontSize, right?

Maybe **MaxFitSize**/**MinFitSize** or **MaxAutoFitSize**/**MinAutoFitSize**
What do you think @jfversluis ?

Reply:
MaxAutoFitSize/MinAutoFitSize would be my preference, seems to be most clear